packages/blockchain-wallet-v4-frontend/src/scenes/Transactions/index.tsx

```diff
@@ -186,6 +186,7 @@ class TransactionsContainer extends React.PureComponent<Props> {
                       width='100px'
                       onClick={() => {
                         this.props.buySellActions.showModal({
+                          cryptoCurrency: coin as CoinType,
```

Comment:
I think that this case that you mention will never happen, because this button is only shown if the coin !== FIAT. In fact, if I go into the transaction screen of any FIAT, the action buttons that I see are Deposit and Withdraw.

Reply:
Could you then `Omit` the FIAT currencies from this type?

Reply:
@pedroapfilho can you clarify what you mean or what I should do? Because if I look at 15 lines above in the same code, CoinType is used in exactly the same way on the same method. So I just copied and pasted the logic of the other button. I mean, the task is very simple.